src/App.vue

```diff
@@ -6,8 +6,8 @@
   <div id="input">
     <div id="username">
       <label>Username: </label>
-      <input type="text" v-model="username">
-      <input id="search" type="button" value="Search" @click=doSomething>
+      <input type="text" v-model="username" @keyup.enter=listRepos>
+      <input id="search" type="button" value="List Repos" @click=listRepos>
```

Comment:
I changed this from "search". Search sounded to vague. 
If "List Repos" doesn't make sense to you, then definitely suggest something better.